### PR TITLE
hw-02

### PR DESCRIPTION
**Pull request by niksaz**

### CODE REVIEW

File: 2/homework/homework.ipynb [hunk 1]
```diff
@@ -0,0 +1,2189 @@
+{
```
Comment (Rauf-Kurbanov) [inline 2/homework/homework.ipynb]: To set seed in pytorch use this
```
    torch.backends.cudnn.deterministic = True
    torch.backends.cudnn.benchmark = False
    torch.manual_seed(seed)
```

Comment (Rauf-Kurbanov) [reply 2/homework/homework.ipynb]: Prefer `.to(device)` instead of `cuda()` since this allows you to use cpu as well and limit your gpu usage

Comment (Rauf-Kurbanov) [reply 2/homework/homework.ipynb]: It is really hard to review with large amounts of code in jupyter
Please explore https://ipython.readthedocs.io/en/stable/config/extensions/autoreload.html

Comment (Rauf-Kurbanov) [reply 2/homework/homework.ipynb]: You do not have any evaluations metrics in your jupyter, how should I evaluate quality of your classifier?

**Review by Rauf-Kurbanov — CHANGES_REQUESTED** [review]

File: 2/homework/homework.ipynb (hunk 1 above)
Comment (niksaz) [reply 2/homework/homework.ipynb]: Thanks for the most up-to-date way of making PyTorch deterministic.

Comment (niksaz) [reply 2/homework/homework.ipynb]: **autoreload** is the feature I though is missing from jupyter. Thank you!

Comment (niksaz) [reply 2/homework/homework.ipynb]: You are right. Test loss is not very representative. Updated to include accuracy and AUC.

**Comment by niksaz** [comment]
@Rauf-Kurbanov, please, have a look once again.

**Comment by Rauf-Kurbanov** [comment]
Now looks nice, thank you
4/4

**Comment by niksaz** [comment]
Thanks for the review, @Rauf-Kurbanov!

**Pull request merged**